pages.hi/common/compare.md

```diff
@@ -0,0 +1,12 @@
+# compare
+
+> 2 छवियों के बीच अंतर देखें।
+> अधिक जानकारी: <https://imagemagick.org/script/compare.php>।
```

Comment:
```suggestion
> अधिक जानकारी: <https://imagemagick.org/script/compare.php>.
```

Reply:
By convention, it would be better to use `purnaviram` or the `। ` symbol instead of `.` while writing in Hindi. It will be easier for native speakers to understand.

Reply:
Understood, although I am not a native speaker I had seen this before in textbooks. Let me ask somebody else regarding this. Also, for future reference leave this conversation as unresolved.

@suyashbhawsar as a native speaker what do you think about this? 

@navarroaxel @CleanMachine1 what do you think about this?

Reply:
@kbdharun Yup, I think it'd be better to use `।` instead of `.` for the pages translated in Hindi.

Reply:
Thanks for the insights, Let's see what others think.

Reply:
Yeah, we use `।` for line endings in Hindi

Reply:
> Yeah, we use `।` for line endings in Hindi

Thanks for the clarification 👍🏽.

Reply:
@kbdharun Do you think using both `।` & `|` would be confusing?

Reply:
> @kbdharun Do you think using both `।` & `|` would be confusing?

There are 2 line endings? 😮

Reply:
> There are 2 line endings? open_mouth

Nope, `|` is the vertical bar we are using to denote `or`.

Reply:
One is Danda `।`, the another is a pipe `|`. Danda is the correct one here.

Reply:
Noted thanks